css-contain-1/Overview.bs

```diff
@@ -23,7 +23,6 @@ spec: css2; urlPrefix: https://www.w3.org/TR/CSS2/
 	type: value; for: content; url: generate.html#value-def-close-quote; text: close-quote
 	type: value; for: content; url: generate.html#value-def-no-open-quote; text: no-open-quote
 	type: value; for: content; url: generate.html#value-def-no-close-quote; text: no-close-quote
-	type: dfn; url: box.html#padding-edge; text: padding edge
```

Comment:
Actually, I think I'd rather keep this one linking to css2, because css-contain is in CR (to be updated soon). To avoid complications around publication request, I think it is best to avoid taking dependencies on a newish spec with non-CR status. Css-box only contains things that were in the CSS2.1 REC, so we could argue it away, but on the flip side, since it doesn't bring any thing new , then there's no rush in updating.